src/main/webapp/repo-bootstrap/global/configuration/samples/sample-blob-stores-config.xml

```diff
@@ -62,36 +62,28 @@
         credentials.secretKey: AWS secret key (optional)
         region: AWS region for the service (optional)
         pathStyleAccess: indicates if path style access should be used for all requests (defaults to false)
-
       -->
 
     <blobStore>
         <id>s3-store</id>
         <type>s3BlobStore</type>
-        <pattern>/static-assets/s3/.*</pattern>
+        <pattern>^/static-assets(?!/(app|css|js|fonts|tracking|seo)/)(.*)</pattern>
         <mappings>
             <mapping>
                 <publishingTarget>preview</publishingTarget>
-                <storeTarget>my-authoring-bucket</storeTarget>
-                <prefix>sandbox</prefix>
-            </mapping>
-            <mapping>
-                <publishingTarget>staging</publishingTarget>
-                <storeTarget>my-authoring-bucket</storeTarget>
-                <prefix>staging</prefix>
+                <storeTarget>${env:PREVIEW_BUCKET_NAME_PATTERN}</storeTarget>
```

Comment:
Since we now have a staging env variable, should we add back the staging target?